veps/vep-1.md

```diff
@@ -0,0 +1,90 @@
+# VEP-1 Vitess Release Cycle
+
+```
+VEP: 1
+Title: Vitess Release Cycle
+Author: Morgan Tocker
+Status: Proposed
+Created: 2019-07-22
+```
+
+## Abstract
+
+Vitess releases have historically been infrequent and not on a fixed release schedule. This has led to a status quo of advanced users maintaining their own branches and pulling in changes from Vitess master.
+
+While keeping _master_ in a permanent stable state is desirable, there will be future scenarios where breaking backward compatibility is desired. This VEP aims to codify a release cycle with rules around backward compatibility, supported upgrade paths, and deprecation schedule.
+
+Definitions of *MUST*, *SHOULD*, *MAY* from [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+## Motivation
+
+The motivation for stable releases is two-fold:
+
+1. Stable releases are easier to consume by novice users. Bug reports, documentation, and support requests will be easier to follow when the user's deployed version is clearer.
+
+2. Advanced users benefit from a clear cycle from which to merge upstream changes into their branch. This also allows for incompatible changes to be introduced at known intervals, which can help improve the health of the project.
+
+## Versioning System
+
+The proposed release cycle is best described as _Monitonic Versioning_ with a release cadence similar to Kubernetes. The terminology `COMPATIBILITY.RELEASE` used in this document is directly sourced from the [Monitonic Versioning Manifesto](http://blog.appliedcompscilab.com/monotonic_versioning_manifesto/):
+
+1. The `COMPATIBILITY` number will be incremented every 12 weeks.
```

Comment:
If there's no breaking change, why do we need a new `compatibility` number?

Reply:
We don't strictly need to. I am sure if we don't have a plan to though, we will just never do it.